core/src/main/java/dev/streamx/cli/util/os/OsConfig.java

```diff
@@ -0,0 +1,16 @@
+package dev.streamx.cli.util.os;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.context.Dependent;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+@Dependent
+public class OsConfig {
+  @ApplicationScoped
+  public OsCommandStrategy strategy(@ConfigProperty(name = "os.name") String osName) {
+    if (osName.contains("win")) {
```

Comment:
what about case? why not ingore case "windows" ? https://github.com/openjdk/jdk/blob/master/src/java.base/windows/native/libjava/java_props_md.c#L481-L568